commands/build.js

```diff
@@ -58,7 +58,8 @@ function buildLumin (argv, indexContent) {
     throw err;
   }
 
-  var buildCommand = `mabu -t ${argv.host ? 'host' : 'device'} ${packagePath}`;
+  var device = argv.debug !== false ? 'device' : 'release_device';
```

Comment:
you can also do `!!debug === true`. This will coerce the `undefined` to false and if it's not `undefined`  `!!` then it's a no-op for a bool

Reply:
or to keep your syntax `!!debug === false`

Reply:
I like @bchapman 's way since double negation is very obscure way to do a type conversion.

Reply:
It's actually a known trick in JS dev

Reply:
> you can also do !!debug === true. This will coerce the undefined to false

This isn't the desired behavior, as `undefined` should default to `true` based on the current goals for the `debug` flag.